engine/execution/ingestion/engine.go

```diff
@@ -1126,7 +1127,7 @@ func (e *Engine) logExecutableBlock(eb *entity.ExecutableBlock) {
 				Int("tx_index", j).
 				Hex("collection_id", logging.ID(col.Guarantee.CollectionID)).
 				Hex("tx_hash", logging.Entity(tx)).
-				Hex("start_state_commitment", eb.StartState).
+				Hex("start_state_commitment", eb.StartState[:]).
```

Comment:
Good suggestion. This PR didn't update the key names though, it kept using the existing ones. It would be great to look at the names consistency in a separate PR maybe.